tt_eager/tt_dnn/op_library/moreh_matmul/moreh_matmul_op.cpp

```diff
@@ -115,30 +85,83 @@ inline Shape compute_output_shape(
     return {Shape(output_shape, padding)};
 }
 
+// Must be provided in the case where an optional output tensor was not provided
+std::vector<Shape> MorehMatmul::compute_output_shapes(const std::vector<Tensor>& input_tensors) const {
+    return {compute_output_shape(input_tensors.at(0).shape(), input_tensors.at(1).shape(), this->transpose_input, this->transpose_other)};
+}
+
+
 inline Tensor create_output_tensor(
```

Comment:
create_output_tensor is no longer used so that it can be deleted.

Reply:
removed!